### ISSUE TITLE
Panel's background don't change after changed theme.

### ISSUE THREAD

**Issue by AkiraVoid** [microsoft/fluentui#22214]
### Environment Information

- **Package version(s)**: Fluent UI React 8.60.2
- **Browser and OS versions**: Microsoft Edge 99.0.1150.46

#### Actual behavior:

When I changed the theme of website by call `loadTheme`, the new theme applied to all components but Panel (Actually, the content in Panel will use new theme but the background will not).

#### Expected behavior:

All part of Panel should use new theme color.

### Priorities and help requested:

Are you willing to submit a PR to fix? (~~Yes~~, No)

Requested priority: (~~Blocking~~, High, ~~Normal~~, ~~Low~~)

Products/sites affected:

**Comment by dbjorge** [comment]
Our team is also able to reproduce this in Accessibility Insights for Web. We noticed this as part of testing an update from fluent UI 8.49.6 to 8.63.1 (https://github.com/microsoft/accessibility-insights-web/pull/5312). We suspect it was introduced by https://github.com/microsoft/fluentui/pull/21466 (released in 8.52.0). It looks like a Chromium rendering issue to me; the components in question are using `background: inherit`, but their inherited background settings aren't getting updated when the parent receives the new background theming. Once the components are forced to rerender (eg, by closing and reopening the Panel), the background color corrects itself.

**Comment by dbjorge** [comment]
[Here is a codepen demonstrating a minimal repro](https://codepen.io/dbjorge/pen/poppxgp) against 8.64.0. It reproduces in 8.52.0 but not 8.51.0.

The repro includes a demonstration of one possible workaround for the issue (setting `background: inherit` on the element with className `ms-Panel-scrollableContent`).

**Comment by dbjorge** [comment]
We'd like to request bumping the priority from "high" to "blocking", since we won't be able to consume a fluentui update in Accessibility Insights for Web until this is resolved.

**Comment by JustSlone** [comment]
@micahgodbolt can you take a look at this, it's a blocker for Accessibility Insights, feel free to rope in @TristanWatanabe too since he worked on the referenced fix

**Comment by Hotell** [comment]
> It looks like a Chromium rendering issue to me

Indeed it's an V8 issues.

works as intended in WebKit, Gecko

**Comment by Hotell** [comment]
I was discussing the possible approach further with @TristanWatanabe and only way how to unblock us is to revert the change that was made.

### Future work

To mitigate the Chromium bug and still maintain proper background coloring in header for scrollable behaviour, we could implement one of following approaches in the future:

1. providing a new style hook that would consistently apply styles to all underlying dom nodes to avoid incosistencies like having header with different color etc.  
2. drop position sticky and replace it with position:fixed and negative margins. This solution would downgrade the UX as additional listener would be needed and on every window change we would need to re-calculate height of header and set the negative margin-top to actual value. 

I'd recommend option 1 for reasons specified
